### DISCUSSION TITLE
Swap Delimiters

### DISCUSSION THREAD

**Discussion by AlainCartiaux** [DaxStudio/DaxStudio#1041] [Q&A] [answered]
Hello
In version 3_0_5 I do not find the option "Swap Delimiters" instead there is the option "Debug Commas" which does not change the delimiter at all.
Is there another way to change the delimiter?
Thanks in advance

**Comment by dgosbell** [comment, marked as answer]
In File > Options under the Advanced tab you can switch off the debug commas setting and it will then re-display the Swap Delimiters option
![image](https://user-images.githubusercontent.com/8241830/215895341-23f09f4d-d909-4702-b1d0-7720735f4c65.png)


In the next release the preferred button out of Swap Delimiters and Debug Commas will stay on the Home tab and the other one will move to the Advanced tab. But at the moment you have to choose which one to display.

**Reply by AlainCartiaux** [reply]
Hello
Thank you very much. Problem solved.
